ui/apps/platform/src/Containers/NetworkGraph/simulation/NetworkPolicySimulatorSidePanel.tsx

```diff
@@ -213,12 +214,7 @@ function NetworkPolicySimulatorSidePanel({
                         <NetworkPoliciesYAML
                             yaml={generatedYaml}
                             additionalControls={[
-                                <Flex
-                                    justifyContent={{ default: 'justifyContentFlexEnd' }}
-                                    alignItems={{ default: 'alignItemsCenter' }}
-                                    spaceItems={{ default: 'spaceItemsNone' }}
-                                    className="pf-v5-u-flex-1"
-                                >
+                                <CodeBlockAction className="pf-v5-u-mr-lg">
```

Comment:
A thought, not necessarily a change request: seeing `Flex` element versus `className` prop, is it possible for fragment in `const actions = (<>…</>);` in CodeViewer.tsx file to become `Flex` element with appropriate props?

Reply:
The parent element for `<CodeBlockAction>` is already display flex, the extra space here was to prevent the help button from being crowded on the right side.
![image](https://github.com/user-attachments/assets/cc5bf1d5-fc85-4f38-a6ae-d1f02cf97ec2)

Although in that case, I suppose it would be more appropriate to have the padding on the button element itself rather than on a higher level layout component.